### ISSUE TITLE
how to run this project locally

### ISSUE THREAD

**Issue by aminabvaal** [Esri/wind-js#37]

**Comment by Jing25** [comment]
I have the same question

**Comment by andresrodrigu** [comment]
Me too

**Comment by on-myown** [comment]
How can I run this project on my local?

**Comment by lozenge0** [comment]
agreed, an example project utilising this would be a huge help in explaining how to use this

**Comment by vitexikora** [comment]
Some tutorial or at least a reproducible repo would be awesome. After several hours, I am unable to run this... I am getting results like that, with the original gfs.json data and almost identical logic as in index.html...
![windy](https://user-images.githubusercontent.com/14636323/81192484-9681cf00-8fba-11ea-9441-44552df5cfb4.png)

**Comment by missionloyd** [comment]
how do you run the project locally?

**Comment by mpayson** [comment]
I tried changing [line 39 in index.html](https://github.com/Esri/wind-js/blob/master/index.html#L39) to remove the slash before "plugins" and it worked for me behind a local static server (eg [serve](https://www.npmjs.com/package/serve)). Does this work for you?
```plugins: location.pathname.replace(/\/[^/]+$/, "") + "plugins"```

**Comment by hbriga** [comment]
1. download the zip file from github or clone the project:  
   git clone https://github.com/Esri/wind-js.git
2. start a minimalist local web-server using python (you can do it using node.js if you prefer), doing this:
    a. on your terminal, change directory to where wind-js folder is located and fire up python. 
    b. type the following:
           import os
           os.system('python3 -m http.server 8765 &')
3. point your browser to localhost:8765
4. you should see the map on your web browser. 

hope this could help.
